### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_DefaultScreen_Keyboard: The keyboard focus shifts to the "968,137" non-interactive control.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2947]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS
Code;#A11ySev3;#WCAG2.1.1;#Benchmark;#DesktopApp;#Win10;#Keyboard;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. TAB to "Microsoft" control.
4. Observe that with next TAB key, keyboard focus shifts to the "968,137" non-interactive control.

### Actual:
The keyboard focus shifts to the "968,137" non-interactive control while navigating using TAB key.

### Expected:
The keyboard focus should not shifts to the "968,137" non-interactive control  while navigating using TAB key.

### User Impact:
The keyboard users will get confused while navigating using TAB key and will think this is interactive control.

**Comment by ejizba** [comment]
An extension cannot control the marketplace page. This issue should be filed on VS Code: https://github.com/microsoft/vscode/issues

**Comment by 12-shweta** [comment]
@ejizba Logged an external bug https://github.com/microsoft/vscode/issues/132744

**Comment by MicroFish91** [comment]
Upstream issue we were tracking against was closed as `by-design`.  Since the extension cannot control the marketplace page, can we have this one revisited and potentially closed out?  Thank you
